services/admin/src/admin/routes/recreate_dataset.py

```diff
@@ -58,8 +57,8 @@ async def recreate_dataset_endpoint(request: Request) -> Response:
             deleted_cached_responses = delete_dataset_responses(dataset=dataset)
             if deleted_cached_responses is not None and deleted_cached_responses > 0:
                 # delete assets
-                delete_asset_dir(dataset=dataset, directory=assets_directory)
-                delete_asset_dir(dataset=dataset, directory=cached_assets_directory)
+                cached_assets_storage_client.delete_dataset_directory(dataset)
+                assets_storage_client.delete_dataset_directory(dataset)
```

Comment:
would be cool to have a test to make sure that it's working fine

Reply:
I added a test for recreate_dataset https://github.com/huggingface/datasets-server/pull/2028, then, I will update it here but using fsspec instead

Reply:
Done